app/api/users.py

```diff
@@ -289,9 +290,9 @@ def before_update_object(self, user, data, view_kwargs):
                     {'source': ''}, "You are not authorized to update this information."
                 )
 
-        if not has_access('is_admin') and data.get('is_verified') is not None:
+        if not has_access('is_admin') and data.get('is_verified') != user.is_verified:
```

Comment:
If a user did not submit `is_verified`, this will throw. This should only throw when user submits `is_verified` and it is different from current one